ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -213,6 +215,13 @@ struct ListCellButtonStyle: ButtonStyle {
                                 bottom: tokens.horizontalCellPadding / 2,
                                 trailing: tokens.horizontalCellPadding))
             .frame(minHeight: height)
-            .background(configuration.isPressed ? Color(tokens.highlightedBackgroundColor) : Color(tokens.backgroundColor))
+            .background(backgroundColor(configuration.isPressed))
+    }
+
+    private func backgroundColor(_ isPressed: Bool = false) -> Color {
```

Comment:
This side effect is still happening on the animation transition of the color.
It's very likely that the background color is a black transparent (#00000000) as opposed to a lighter color like white transparent (#FFFFFF00).

We need to either animate it to white transparent or just animate the alpha value to avoid this darkening animation as the color fades away:


https://user-images.githubusercontent.com/68076145/113211492-61ade480-922a-11eb-879d-5926ab207d6e.mov

Reply:
Let me check if those value work in dark mode and I will update the new effect.

Reply:
I updated the background color. These colors are updated on the client side and not in the library. 
Below is the preview of animation in light and dark mode.

https://user-images.githubusercontent.com/63682282/113232364-d9dbd080-9251-11eb-8d4f-c13ae715074e.mov
https://user-images.githubusercontent.com/63682282/113232343-d0526880-9251-11eb-8756-2adea024b1e7.mov

Reply:
This is now updated with using drawers horizontal background color

https://user-images.githubusercontent.com/63682282/113598494-101ea480-95f2-11eb-9f56-d12dc02ffc04.mov